Developers/Newcomers_Guide.rst

```diff
@@ -127,24 +122,57 @@ into your bash. This will analyze your code and help you fix it.
 
     `Executing tests <http://coala.readthedocs.io/en/latest/Developers/Executing_Tests.html>`_
 
-Step 5. Sending Your Changes
+Step 5. Create Branch and Commit Changes
+----------------------------------------
+
+When you are happy with your code, you should do two things.
+
+First create new branch so your changes are separated.
+
+::
+
+    $ git checkout -b my-fixed-issue
+
+Where ``my-fixed-issue`` is some arbitrary name to help you associate
+this issue with your changes.
+
+Second part is commit your changes to this branch. For this you need to
+prepare changes files to commit and then commit them with commit message.
```

Comment:
prepare the files you changed by adding them to the commit, and then commiting them with a message.